api/swagger.yaml

```diff
@@ -2833,24 +2854,38 @@ definitions:
         type: "object"
         additionalProperties:
           type: "string"
+        example:
+          com.example.some-label: "some-value"
+          com.example.some-other-label: "some-other-value"
       Data:
-        description: "Base64-url-safe-encoded secret data"
-        type: "array"
```

Comment:
FWIW I changed this from an array to just string; I guess this was because in go it's a `[]byte`

<img width="990" alt="screen shot 2017-07-27 at 23 53 47" src="https://user-images.githubusercontent.com/1804568/28693824-fd711a76-7326-11e7-9674-8c9cffaa6a48.png">